client/themes/zenburn.css

```diff
@@ -134,29 +133,35 @@ body {
   border-color: #101010;
 }
 
-#form .input {
-  margin-right: 0;
-}
-
 #form #input {
   background-color: #434443;
   border-color: #101010;
   color: #dcdccc;
-  padding-left: 1em !important;
```

Comment:
Well, it had to be important because javascript sets padding-left directly on the element :D